docs/2_2_managing.md

```diff
@@ -0,0 +1,16 @@
+# 🧰 Managing patched applications
+
+After patching an application, you may want to manage it. This page will guide you through managing patched applications.
+
+## 🪜 Steps to manage patched applications
+
+1. Tap on the **Dashboard** tab in the bottom navigation bar
+2. Select the **Installed** chip
+3. Tap on the **Info** button for the application you want to manage.
+4. Chose one of the options in the menu that appears to manage the selected application.
```

Comment:
```suggestion
4. Choose a managing action from the menu
```

Reply:
Not sure now that I think of it, management action works? But could also be plainly "Choose one of the options from the menu"